app/lib/server/functions/deleteRoom.js

```diff
@@ -4,6 +4,7 @@ import { callbacks } from '../../../callbacks';
 export const deleteRoom = function(rid) {
 	Messages.removeFilesByRoomId(rid);
 	Messages.removeByRoomId(rid);
+	callbacks.run('beforeDeleteRoom', rid);
```

Comment:
why not pass the hole room here, same to `afterDeleteRoom`, we could save some finds here in the future